packages/dd-trace/src/plugins/util/git.js

```diff
@@ -36,8 +36,17 @@ function isShallowRepository () {
 }
 
 function unshallowRepository () {
-  sanitizedExec('git', ['config', 'remote.origin.partialclonefilter', '"blob:none"'])
-  sanitizedExec('git', ['fetch', '--shallow-since="1 month ago"', '--update-shallow', '--refetch'])
```

Comment:
run `git version` here and check if the version is >=2.27. If it's older than 2.27, we don't run the following commands